rust/lance/src/index/vector/ivf/v2.rs

```diff
@@ -692,9 +694,9 @@ mod tests {
     }
 
     #[rstest]
-    #[case(4, DistanceType::L2, 0.9)]
-    #[case(4, DistanceType::Cosine, 0.9)]
-    #[case(4, DistanceType::Dot, 0.8)]
+    #[case(4, DistanceType::L2, 0.75)]
```

Comment:
You mentioned the new algorithm can have decent recall? Should we bump this up

Reply:
fixed